RELEASE.md

```diff
@@ -0,0 +1,123 @@
+# Release Documentation
+
+This document contains Robotest's API stability & versioning guarantees
+as well as instructions to help maintainers meet these guarantees and
+ensure consistent releases.
+
+# API Stability
+Robotest follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
+(semver) for all image tags with a version specifier (for example `2.0.0`).
+Named tags (for example `gce-stable`) offer no stability promises and may
+change at any point.
+
+Robotest's command line flags and configuration options logs are
+it's [semver public API](https://semver.org/spec/v2.0.0.html#spec-item-1).
+This includes configuration via environment variables in the
+[run_suite.sh](./docker/suite/run_suite.sh) helper script.
+
+Notably: console output, cloud logs, and error messages are not part of
+Robotest's public API and may change in minor releases.
+
+All go APIs (exported or not) are not part of Robotest's public API and may
+change at any time.
+
+These stability guarantees specifically target's Gravity's CI use of Robotest.
+
+Prior to 2.0.0, Robotest broke semver rules, with both new features and
+backwards incompatible changes added as patch releases.  Here be dragons. :dragon:
+
+# Release Instructions
+
+## Prepare
+
+### Select the commit that will become the release.
+
+Checkout the branch/commit that will become the release.
+
+```
+# git checkout master
+```
+
+### Check that changes are within the semver risk profile.
+
+Use `git log` to check that all changes have <= the appropriate risk for the
+planned release bump (major, minor, patch). Each commit in the release should
+be traceable to a PR with the "Risk Profile" filled out.
+
+Be sure to check beyond all [pre-release](https://semver.org/spec/v2.0.0.html#spec-item-9)
```

Comment:
Nit: Having a bit of a hard time understanding this sentence.

Reply:
I reworded this 0028ed3.  It is still kinda clunky though.  I'm open to suggestions.